discord/client.py

```diff
@@ -1153,3 +1153,133 @@ def transform(d):
         """
         data = await self.http.get_webhook(webhook_id)
         return Webhook.from_state(data, state=self._connection)
+
+    async def edit_settings(self, **kwargs):
+        """|coro|
+
+        Edits the client user's settings. (Only available to selfbots)
```

Comment:
```suggestion
        Edits the client user's settings. (Only applicable to user accounts.)
```
a la [`User.edit`](https://github.com/Rapptz/discord.py/blob/d48a424f6f464044a59275d5e885b493a4a1aef7/discord/user.py#L351).